src/android/com/pbakondy/SpeechRecognition.java

```diff
@@ -76,6 +77,12 @@ public void run() {
     });
   }
 
+  @Override
+  public void onStop() {
+    super.onStop();
+    callbackContext.error(BACKGROUND_ERROR);
```

Comment:
error only if it's running ?

Reply:
@azharbeebeejaun Yes if there is an instantiated SpeechRecogniser class, it will then trigger.

Reply:
Helpful: https://github.com/distinctdan/cordova-plugin-extra-events

Reply:
Removed Use [cordova-plugin-extra-events](https://github.com/distinctdan/cordova-plugin-extra-events) instead